service/controller/app/resource/releasemigration/resource.go

```diff
@@ -61,20 +61,38 @@ func (r *Resource) Name() string {
 	return Name
 }
 
-func (r *Resource) findHelmV2Releases(k8sClient kubernetes.Interface, tillerNamespace string) ([]string, error) {
+func (r *Resource) findHelmV2Releases(k8sClient k8sclient.Interface, tillerNamespace string) ([]string, error) {
+	charts := make(map[string]bool)
+
+	// Get list of chart CRs as not all helm 2 releases will have a chart CR.
+	list, err := k8sClient.G8sClient().ApplicationV1alpha1().Charts("giantswarm").List(metav1.ListOptions{})
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
```

Comment:
In the control plane there are also Helm 2 releases managed by `draughtsman` and `opsctl ensure appcatalogs`.

So we should only migrate helm releases that have a chart CR.